src/libs/routes/errorHandler.ts

```diff
@@ -0,0 +1,10 @@
+const errorHandler = ((err, req, res, next) => {
+    console.log('Error', err);
+    res.send({
+        error: 'not found',
```

Comment:
add an optional error if we don't get any like:
 error: error || 'undefined'

Reply:
Done